openfl/component/ca/ca.py

```diff
@@ -27,6 +28,26 @@
 CA_CONFIG_JSON = Path('config/ca.json')
 
 
+def get_system_and_architecture():
+    """Get system and architecture of machine."""
+    uname_res = platform.uname()
+    system = uname_res.system.lower()
+
+    architecture_aliases = {
+        ('x86_64', ): 'amd64',
+        ('armv6l', ): 'armv6',
+        ('armv7l', ): 'armv7',
+        ('aarch64', ): 'arm64'
+    }
+    architecture = uname_res.machine
+    for aliases in architecture_aliases:
+        if architecture in aliases:
+            architecture = architecture_aliases[aliases]
+            break
+
+    return system, architecture
```

Comment:
I think that we need to have some default architecture here (e.g. `amd64`) because in case if we didn't match architecture with alias wrong value will be returned. Or maybe we need to raise exception in this case.

Reply:
Arch might be correctly detected without alias, for example, on arm variants. Exception for case with no applicable binaries was added